### PR TITLE
Created the service component

### PR DESCRIPTION
**Pull request by bayanbamb**

### CODE REVIEW

File: src/app/components/charts/pie-grid/pie-grid.component.ts
Comment (MohammadWS) [inline src/app/components/charts/pie-grid/pie-grid.component.ts:1]: a PR shouldn't have commented code

File: src/app/events.service.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+@Injectable({
+  providedIn: 'root'
+})
+export class EventsService {
+  private categoryUrl = 'http://localhost:3000/event-categories';
+
+  constructor(private http: HttpClient) { }
+
+  getEventCategories(): Observable<any[]> {
+    return this.http.get<any[]>(this.categoryUrl);
```
Comment (MohammadWS) [inline src/app/events.service.ts]: it's better to avoid using any if possible

**Review by MohammadWS — COMMENTED** [review]
Looks good but need a little modification

File: src/app/events.service.ts (hunk 1 above)
Comment (bayanbamb) [reply src/app/events.service.ts]: Changed it to interface👍🏻

**Review by MohammadWS — APPROVED** [review]
Looks great!

**Pull request merged**